code/osx32_start.S

```diff
@@ -8,7 +8,7 @@
 
 .intel_syntax noprefix
 .globl _start
-.globl syscall0, syscall1, syscall2, syscall3, syscall4, syscall5, syscall6
+.globl _syscall0, _syscall1, _syscall2, _syscall3, _syscall4, _syscall5, _syscall6
```

Comment:
Interesting that clang does this name mangling stuff and g++ doesn't https://stackoverflow.com/a/5908708

Might be nice to leave a comment that says clang needs underscores to link properly.

Reply:
Updated via 8e80d67